### PR TITLE
Changed SMG Weapon Category (#317)

### PR DESCRIPTION
**Pull request by jmartinez989**
SMGs were getting the damage bonus from the assault rifle weapon
breakthrough. This was because their weapon category was 'rifle' which
is the same category as the assualt rifle. Changed the weapon category
to 'smg' and added the needed lines in XComClassData.ini to allow the
classes that could equipem them to continue to equip them.

### CODE REVIEW

File: LongWarOfTheChosen/Src/LW_LaserPack_Integrated/Classes/X2Item_LaserWeapons.uc [hunk 1]
```diff
@@ -6,8 +6,8 @@
 //---------------------------------------------------------------------------------------
 class X2Item_LaserWeapons extends X2Item config(GameData_WeaponData);
 
-//�Variables�from�config�-�GameData_WeaponData.ini
-//�*****�Damage�arrays�for�attack�actions��*****
+//�Variables�from�config�-�GameData_WeaponData.ini
+//�*****�Damage�arrays�for�attack�actions��*****
```
Comment (pledbrook) [inline LongWarOfTheChosen/Src/LW_LaserPack_Integrated/Classes/X2Item_LaserWeapons.uc]: What's this?

Comment (jmartinez989) [reply LongWarOfTheChosen/Src/LW_LaserPack_Integrated/Classes/X2Item_LaserWeapons.uc]: I do not know how this happened. Somehow my editor replaced the spaces with unprintable characters.

I'll get this fixed.

Comment (jmartinez989) [reply LongWarOfTheChosen/Src/LW_LaserPack_Integrated/Classes/X2Item_LaserWeapons.uc]: This is fixed now. I'll make sure to keep an eye for this again.

**Comment by WilleBack** [comment]
This does mean that there is now no Weapon Breakthrough for the SMG, so one should probably be created if the breakthroughs are to be retained.

**Comment by jmartinez989** [comment]
> This does mean that there is now no Weapon Breakthrough for the SMG, so one should probably be created if the breakthroughs are to be retained.

We do not yet even know if we are keeping any of the breakthroughs. Maybe the tier specific breakthroughs we will but the weapon type breakthroughs might receive the axe.

**Pull request merged**